src/Giraffe/Giraffe.fsproj

```diff
@@ -41,23 +41,29 @@
       <PackagePath>$(PackageIconUrl)</PackagePath>
     </None>
   </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' != 'netcoreapp3.0' ">
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.2.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="2.2.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.*" />
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.0.0" />
```

Comment:
Should this be version `2.2.*`, as I assume `3.0.0` was to fix compatibility with `netcoreapp3.0` which would now be covered by the framework reference.